client/tests/acceptance/user-can-click-rwcds-form-edit-test.js

```diff
@@ -86,4 +86,29 @@ module('Acceptance | user can click rwcds edit', function(hooks) {
     await fillIn('[data-test-textarea="dcpSitehistory"]', exceedMaximum(600, 'String'));
     assert.dom('[data-test-validation-message="dcpSitehistory"').hasText('Text is too long (max 600 characters)');
   });
+
+  test('Validation messages display for Proposed Actions', async function(assert) {
+    this.server.create('project', 1, {
+      packages: [this.server.create('package', 'applicant', 'rwcdsForm')],
+    });
+
+    await visit('/rwcds-form/1/edit');
+
+    assert.equal(currentURL(), '/rwcds-form/1/edit');
+
+    await fillIn('[data-test-input="dcpModifiedzrsectionnumber"]', exceedMaximum(25, 'String'));
```

Comment:
I really like this helper you made, `exceedMaximum` — it's simple and it's name so that it explains itself pretty well